### PR TITLE
LPS-85697 Test assetTypes.

### PR DESCRIPTION
**Pull request by cgoncas**

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

File: modules/apps/headless/headless-foundation/headless-foundation-test/src/testIntegration/java/com/liferay/headless/foundation/resource/v1_0/test/VocabularyResourceTest.java [hunk 1]
```diff
@@ -102,4 +113,42 @@ protected Vocabulary testPutVocabulary_addVocabulary() throws Exception {
...
+			}
+
+			if (Objects.equals(
+					assetType1.getRequired(), assetType2.getRequired()) &&
+				Objects.equals(assetType1.getType(), assetType2.getType()) &&
+				Objects.equals(
+					assetType1.getSubtype(), assetType2.getSubtype())) {
+
+				continue;
+			}
```
Comment (ruben-pulido) [inline modules/apps/headless/headless-foundation/headless-foundation-test/src/testIntegration/java/com/liferay/headless/foundation/resource/v1_0/test/VocabularyResourceTest.java]: Aren't we missing an `else {return false;}` here?

Comment (cgoncas) [reply modules/apps/headless/headless-foundation/headless-foundation-test/src/testIntegration/java/com/liferay/headless/foundation/resource/v1_0/test/VocabularyResourceTest.java]: Yeahhh :)

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:relevant

**Pull request closed without merging**